Scripts/CherwellIncidentOwnTask/CherwellIncidentOwnTask.py

```diff
@@ -0,0 +1,48 @@
+from CommonServerPython import *
+
+args = demisto.args()
+
+# ####################################################################################
+# ############################## CONFIGURATION PART ##################################
+# ####################################################################################
+
+"""
+`RELATIONSHIP_ID` is the id of the relationship you wish to base your link on. In this case we wanted to create script
+for incidents to own tasks, thus, we used the "Incident Owns Task" relationship id.
+"""
+RELATIONSHIP_ID = '9369187528b417b4a17aaa4646b7f7a78b3c821be9'
```

Comment:
put this as an exception for guard-rails

Reply:
- [x] Check